Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/FarePolicy.hs

```diff
@@ -288,6 +313,19 @@ mkFarePolicyBreakups mkValue mkBreakupItem mbDistance mbTollCharges farePolicy =
               perExtraKmStepFareItem = mkBreakupItem perExtraKmStepFareCaption (mkValue $ highPrecMoneyToText s1.perExtraKmRate)
           mkPerExtraKmStepFareItem (perExtraKmStepFareItems <> [perExtraKmStepFareItem]) (s2 : ss) baseDistance
 
+        mkPerMinStepFareItem perMinStepFareItems [] = perMinStepFareItems
+        mkPerMinStepFareItem perMinStepFareItems [fp1] = do
+          let sectionStartDuration = fp1.rideDurationInMin
+              perMinStepFareCaption = show $ Tags.PER_MIN_STEP_FARE sectionStartDuration Nothing
```

Comment:
Lets name it PER_MINUTE_STEP_FARE, MIN is getting confused with mininum.

Reply:
Fixed.